docs/get-started/getting-started-databricks.md

```diff
@@ -15,10 +15,14 @@ on NVIDIA GPUs on Databricks.
     * AWS: 7.3 LTS ML (GPU, Scala 2.12, Spark 3.0.1)
     * Azure: 7.3 LTS ML (GPU, Scala 2.12, Spark 3.0.1)
 
+Databricks 7.3 LTS ML runs CUDA 10.1 Update
+2[*](https://docs.databricks.com/release-notes/runtime/7.3ml.html#system-environment), and the
```

Comment:
The hyperlink on the asterisk is so small it will be easily missed.  Maybe make this more obvious?
```suggestion
[Databricks 7.3 LTS ML](https://docs.databricks.com/release-notes/runtime/7.3ml.html#system-environment)
runs CUDA 10.1 Update 2, and the
```

Reply:
I'm a little confused by this line, how does this compare tot he prequisite section?    we link to their page so it is saying use those init scripts?

Reply:
During a previous virtual review (0.2 timeframe) a user tried a different version of the runtime, and ended up using the wrong cudf version (10.2 or 11.0) because they were not matching the CUDA version.  Even though we are not officially supporting other versions of Databricks, I added this text so as to make sure people were aware of that Databricks has a specific version of CUDA they support (10.1 as of 7.3 LTS ML).  

I can remove this if it is confusing.

Reply:
I see, that makes sense, I was trying to read more into then needed.

Reply:
Updated hyperlink per suggestion.